ui/main/src/app/modules/admin/components/table/perimeters-table.component.ts

```diff
@@ -49,16 +49,44 @@ export class PerimetersTableComponent extends AdminTableDirective implements OnI
             sortable: false,
             filter: 'agTextColumnFilter',
             filterParams: {
-                valueGetter: (params) => {
+                textMatcher: (params) => {
                     const currentProcessDef = this.processesDefinition.filter(
                         (processDef) => processDef.id === params.data.process
                     )[0];
                     let text = '';
                     params.data.stateRights.forEach((stateRight) => {
                         if (currentProcessDef.states.get(stateRight.state))
-                            text += currentProcessDef.states.get(stateRight.state).name + ' ';
+                            text += currentProcessDef.states.get(stateRight.state).name.toLocaleLowerCase() + ' ';
                     });
-                    return text;
+
+                    let response = false;
+
+                    switch (params.filterOption) {
+                        case 'equals':
+                            response = text.toLocaleLowerCase() === params.filterText;
```

Comment:
When I test with "equals" "message" I have no result. It should return at least the perimeter with id=externalRecipent no ?